rllib/utils/pre_checks/env.py

```diff
@@ -24,10 +28,15 @@ def check_env(env: EnvType) -> None:
             env, (BaseEnv, gym.Env, MultiAgentEnv, RemoteBaseEnv, VectorEnv)):
         raise ValueError(
             "Env must be one of the supported types: BaseEnv, gym.Env, "
-            "MultiAgentEnv, VectorEnv, RemoteVectorEnv")
+            "MultiAgentEnv, VectorEnv, RemoteBaseEnv")
 
-    if isinstance(env, gym.Env):
+    if isinstance(env, gym.Env) and not isinstance(env, MultiAgentEnv):
```

Comment:
wait, is there any update to this condition?
I had another comment suggesting that we should simply:

```
if isinstance(env, gym.Env):
    check_gym_environments(env)
if isinstance(env, MultiAgentEnv):
    check_multiagent_environments(env)
if isinstance(env, VectorEnv) or isinstance(env, RemoteBaseEnv):
    logger.warning("not implemented")
```

let me know if you don't think we should do it this way.

Reply:
Yeah, this makes sense to restructure the if-block as above. ^^ I can do this now ...

Reply:
I see, Jun's suggestion would break the observation space checks.
Let's leave it like this for now.

```
if isinstance(env, MultiAgentEnv):
    check_multiagent_environments(env)
elif isinstance(env, gym.Env):
    check_gym_environments(env)
elif isinstance(env, VectorEnv) or isinstance(env, RemoteBaseEnv):
    logger.warning("not implemented")
```

Reply:
just curious at this point, if it's going to break the observation space check, how can it still claim to be a gym.Env?